### PR TITLE
Enhance the editor experience

### PR DESCRIPTION
**Pull request by XVincentX**
This PR will basically solve those tho issues:

#166 
#167

### Dig into details

1. The `textarea` has been replaced by [ace](http://ace.c9.io), text mode, github theme.This will allow a better editing experience. Of course, I removed the `Tabbed` extension.

Whenever is possible, I am forwarding parser errors into the editor:
image

2. The way the object are getting parsed is slightly different. Previously we were using a fake APIBlueprint where attributes where injected. In this way we would never be able to render references to other structures. Now, on the other side, we're using `protagonist` to parse directly the `#Data Structures` part of APIBlueprint. This involved the change in the way [mson-zoo](https://github.com/apiaryio/mson-zoo) are server. Now, we have to preamble them with the `# Data Structure` header. The same goes with rendering tests.

3. A sample MSON object is now inserted and rendered when you open the playground.

Next step: render multiple data structures per time.

### CODE REVIEW

**Comment by Baggz** [comment]
@XVincentX 

No syntax highlighting? :cry:

File: scripts/generateFixtures.js [hunk 1]
```diff
@@ -17,7 +17,13 @@ if (!fs.existsSync(fixtureLocation)) {
 
 msonZoo.samples.forEach((sample) => {
 
-  parseMson(sample.code, (err, result) => {
+  let header = dedent`
+    # Data Structures
+
+    ## Mson Struct
```
Comment (Baggz) [inline scripts/generateFixtures.js]: _**Nit:**_ MSON, not Mson.

File: package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "build:client:full": "node ./scripts/run.js ./clientBuild full",
     "build:client:noDep": "node ./scripts/run.js ./clientBuild noDep",
     "generate-fixtures": "npm run build:server && node ./scripts/run.js ./generateFixtures",
-    "lint": "eslint src playground --ignore-pattern playground/components/TabOverride.js",
+    "lint": "eslint src playground",
```
Comment (Baggz) [inline package.json]: :+1:

**Comment by XVincentX** [comment]
@Baggz Syntax highlight: is there an easy way to do that? We should write rules and so on...or am I missing something?

**Comment by Baggz** [comment]
@XVincentX 

I would apply just basic Markdown syntax highlighting.

**Comment by XVincentX** [comment]
On the way.

**Pull request merged**